orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -179,23 +179,23 @@ def _create_training_session(self):
         # 0:Verbose, 1:Info, 2:Warning. 3:Error, 4:Fatal. Default is 2.
         session_options.log_severity_level = 1
```

Comment:
How about changing log level to 2. Too much logs confuse new comers to experiment on ORTModule

Reply:
ok